contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -290,7 +295,7 @@ contract KeeperRegistry is
    * @param id upkeep to fund
    * @param amount number of LINK to transfer
    */
-  function addFunds(uint256 id, uint96 amount) external override onlyActiveUpkeep(id) {
+  function addFunds(uint256 id, uint96 amount) external override whenNotPaused onlyActiveUpkeep(id) {
```

Comment:
Wondering if we need this? When contract is paused, check/perform upkeep can be paused but i don't see an issue in allowing users to add / withdraw funds. 

Especially since we are running out of contract size

Reply:
removed